config/wom.js

```diff
@@ -1,7 +1,8 @@
 const { WOMClient } = require('@wise-old-man/utils');
 const { Environment } = require('../services/environment');
 
-const WOM_RATE_LIMIT = 100;
+const WOM_RATE_LIMIT = Environment.WOM_API_KEY ? 100 : 20;
+const TIME_PER_TOKEN = 60000 / WOM_RATE_LIMIT;
```

Comment:
I would rename this slightly to make explicit that this is time expressed in milliseconds, so something like `TIME_PER_TOKEN_IN_MS` or `TIME_PER_TOKEN_MS` would suffice

Reply:
Changed in 6a5197c